suite/smoke/multi_network_contracts_test.go

```diff
@@ -29,7 +29,7 @@ var _ = Describe("Basic Contract Interactions @contract", func() {
 		By("Deploying the environment", func() {
 			var err error
 			suiteSetup, err = actions.MultiNetworkSetup(
-				environment.NewChainlinkCluster(0),
+				environment.NewChainlinkCluster(1),
```

Comment:
Is this necessary? This test only makes sure that contracts are launched, with no interaction with chainlink nodes.

Reply:
If it's zero, the test fails with 
E1109 22:03:11.641977   12154 portforward.go:400] an error occurred forwarding 45191 -> 9417: error forwarding port 9417 to pod 30a423d36711bfd5d1f1110b27de89c052d3e0101c81463cee30c141443a2b31, uid : exit status 1: 2021/11/09 22:03:11 socat[16410] E connect(5, AF=2 127.0.0.1:9417, 16): Connection refused

Reply:
I think it has something to do with this change
https://github.com/smartcontractkit/integrations-framework/pull/138/files#diff-dedb54e5426283c5c6cefbe9a1587207f8e8060396acd9e2914b6194630f61f7R679

But didn't dig deeper to find out why

Reply:
Also hopefully the env changes that @skudasov is making will fix this

Reply:
Wouldn't it make sense to just move the mock server manifests into the `basicDependencyGroup`?

Reply:
good idea